source/JsMaterialX/JsMaterialXFormat/JsXmlIo.cpp

```diff
@@ -4,6 +4,7 @@
 //
 
 #include "../Helpers.h"
+#include "./emscriptenTypeRegistration.h"
```

Comment:
Can we rename this into something more specific, like `FileSearchPathHelper`? Note that it should also follow the `UpperCamelCase` style.